cpu/stm32_common/Makefile.dep

```diff
@@ -5,3 +5,9 @@ ifneq (,$(filter periph_i2c,$(USEMODULE)))
     USEMODULE += periph_i2c_2
   endif
 endif
+
+# flashpage and eeprom periph implementations share flash lock/unlock functions
+# in periph_flash_common
+ifneq (,$(filter periph_flashpage periph_eeprom,$(FEATURES_REQUIRED)))
+  USEMODULE += periph_flash_common
+endif
```

Comment:
Well, this is perfect, but now it comes the annoying part... can you split this into a separated PR? BTW @cladmi just by chance aren't you dealing with this in one of your bunch of PRs?

Reply:
See #10146